test/jdk/java/util/regex/RegExTest.java

```diff
@@ -4557,7 +4553,61 @@ public static void badIntersectionSyntax() {
         assertTrue(e.getMessage().contains("Bad intersection syntax"));
     }
 
-    //This test is for 8281315
+    //This test is for 8264160
+    public static void wordBoundaryInconsistencies() {
```

Comment:
There should be an @Test I believe as unless I missed it, this method is not called anywhere else